### PR TITLE
dev, v2.1, v3.0: fix broken links

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Fix broken links related to refactoring.

### Which version does your change affect? 

dev, v2.1, v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

@lilin90 PTAL

### CODE REVIEW

File: dev/how-to/maintain/backup-and-restore.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ category: how-to
 
 # 备份与恢复
 
-本文档将详细介绍如何对 TiDB 进行全量备份与恢复。增量备份与恢复可使用 [TiDB Binlog](/tools/binlog/overview.md)。
+本文档将详细介绍如何对 TiDB 进行全量备份与恢复。增量备份与恢复可使用 [TiDB Binlog](/reference/tools/binlog/overview.md)。
```
Comment (lilin90) [inline dev/how-to/maintain/backup-and-restore.md]:
```suggestion
本文档将详细介绍如何对 TiDB 进行全量备份与恢复。增量备份与恢复可使用 [TiDB Binlog](/reference/tidb-binlog-overview.md)。
```

File: v3.0/how-to/migrate/from-aurora.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ DM 在增量同步阶段依赖 `ROW` 格式的 binlog，如果未启用 binlog
```
Comment (lilin90) [inline v3.0/how-to/migrate/from-aurora.md]: Please also update the link in line 21 and then apply the same changes to `dev` and `v2.1`.

File: dev/how-to/scale/horizontally.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ TiDB 集群可以在不影响线上服务的情况下动态进行扩容和缩容
 
 下面分别介绍如何增加或者删除 PD，TiKV 以及 TiDB 的节点。
 
-下面用到的 pd-ctl 文档可以参考 [pd-control](/tools/pd-control.md)。
+下面用到的 pd-ctl 文档可以参考 [pd-control](/reference/tools/pd-control.md)。
```
Comment (lilin90) [inline dev/how-to/scale/horizontally.md]: Please also update this link in `v3.0` and `v2.1`.

Comment (CaitinChen) [reply dev/how-to/scale/horizontally.md]: @lilin90 This dead link only exists in `dev`.

Comment (lilin90) [reply dev/how-to/scale/horizontally.md]: @CaitinChen I just checked. They do exist in `v3.0` and `v2.1`. Take another look?

Comment (CaitinChen) [reply dev/how-to/scale/horizontally.md]: @lilin90 Thanks for your check! Updated~

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

File: v3.0/how-to/migrate/from-aurora.md (hunk 2 above)
Comment (CaitinChen) [reply v3.0/how-to/migrate/from-aurora.md]: @lilin90 Fixed in https://github.com/pingcap/docs-cn/pull/1569

Comment (lilin90) [reply v3.0/how-to/migrate/from-aurora.md]: OK.

**Pull request merged**